sdk/datalake/azure-mgmt-datalake-analytics/MANIFEST.in

```diff
@@ -1 +1,6 @@
+recursive-include tests *.py *.yaml
```

Comment:
Why is this file in the PR?

Reply:
Unfortunately some CI process messed up with this PR. Could you ignore all non-python files?

Reply:
I mean, only the python files under /eventprocessor

Reply:
I reverted the three commits made by AutorestCI. Now the code is clean